core_lib_generator/core_lib_config_generate_yaml.py

```diff
@@ -32,6 +32,7 @@ def _get_env_variables(data):
 data_layers.setdefault('data', [])
 data_layers.setdefault('data_access', [])
 data_layers.setdefault('service', [])
+server_type = 3

```

Comment:
3 is a magic number 
https://chatgpt.com/share/66ea8bf4-dfd8-800c-bd68-c50d3973c02d